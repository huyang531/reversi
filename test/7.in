cd
cc
dc
ec
fd
ce
eb
fe
ff
fa
ea
da
ef
fb
cb
gd
gc
fc
db
hc
bd
be
bc
cf
af
ca
bf
ge
cg
ac
df
ad
he
hf
hd
gf
fg
ae
ab
eg
dh
eh
fh
dg
gb
gg
ga
hg
bb
ha
hb
ch
hh
gh
bg
ah
bh
ag
aa
ba
