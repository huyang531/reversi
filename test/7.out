  abcdefgh
a ........
b ........
c ........
d ...WB...
e ...BW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ...B....
d ...BB...
e ...BW...
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WB....
d ...WB...
e ...BW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WB....
d ..BBB...
e ...BW...
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WB....
d ..WBB...
e ..WWW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WB....
d ..WBB...
e ..WBW...
f ...B....
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e ..WBW...
f ...B....
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e .BBBW...
f ...B....
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e .BBWW...
f ...BW...
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e .BBWW...
f ...BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e .WBWW...
f W..BBB..
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e BBBWW...
f W..BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d W.WWW...
e WBBWW...
f W..BBB..
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d W.WWW...
e WBBBBB..
f W..BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d W.WWW...
e WBWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c .BWWW...
d W.BWW...
e WBWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c .BWWW...
d W.BWW...
e WBWWBB..
f WW.WBB..
g ...W....
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c .BWWW...
d W.BWW...
e WBWWBB..
f WW.BBB..
g ..BW....
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c .BWWW...
d W.BWW...
e WWWWBB..
f WWWBBB..
g ..BW....
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c .BWWW...
d WBBWW...
e WWBWBB..
f WWWBBB..
g ..BW....
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c .BWWW...
d WBBWW...
e WWBWBB..
f WWWBBB..
g ..WW....
h ..W.....
Enter move for colour B (RowCol):   abcdefgh
a ........
b ...B....
c .BBBW...
d WBBBW...
e WWBBBB..
f WWWBBB..
g ..WW....
h ..W.....
Enter move for colour W (RowCol):   abcdefgh
a ........
b ...BW...
c .BBWW...
d WBWBW...
e WWBBBB..
f WWWBBB..
g ..WW....
h ..W.....
Enter move for colour B (RowCol):   abcdefgh
a ........
b ..BBW...
c .BBBW...
d WBWBB...
e WWBBBB..
f WWWBBB..
g ..WW....
h ..W.....
Enter move for colour W (RowCol):   abcdefgh
a ........
b ..BBW...
c .BBBWW..
d WBWBW...
e WWBWBB..
f WWWBBB..
g ..WW....
h ..W.....
Enter move for colour B (RowCol):   abcdefgh
a .....B..
b ..BBB...
c .BBBWW..
d WBWBW...
e WWBWBB..
f WWWBBB..
g ..WW....
h ..W.....
Enter move for colour W (RowCol):   abcdefgh
a .....B..
b ..BBB...
c WWWWWW..
d WBWBW...
e WWBWBB..
f WWWBBB..
g ..WW....
h ..W.....
Enter move for colour B (RowCol):   abcdefgh
a .....B..
b ..BBBB..
c WWWWBW..
d WBWBW...
e WWBWBB..
f WWWBBB..
g ..WW....
h ..W.....
Enter move for colour W (RowCol):   abcdefgh
a .....B..
b ..BBBB..
c WWWWBW..
d WWWBW...
e WWWWWB..
f WWWWWB..
g ..WWW...
h ..W.....
Enter move for colour B (RowCol):   abcdefgh
a .....B..
b ..BBBB..
c WWWWBBB.
d WWWBW...
e WWWWWB..
f WWWWWB..
g ..WWW...
h ..W.....
Enter move for colour W (RowCol):   abcdefgh
a ..W..B..
b ..WBBB..
c WWWWBBB.
d WWWBW...
e WWWWWB..
f WWWWWB..
g ..WWW...
h ..W.....
Enter move for colour B (RowCol):   abcdefgh
a ..W..B..
b ..WBBB..
c WWWWBBB.
d WWWBBB..
e WWWWWB..
f WWWWWB..
g ..WWW...
h ..W.....
Enter move for colour W (RowCol):   abcdefgh
a ..WW.B..
b ..WWBB..
c WWWWBBB.
d WWWBBB..
e WWWWWB..
f WWWWWB..
g ..WWW...
h ..W.....
Enter move for colour B (RowCol):   abcdefgh
a ..WW.B..
b ..WWBB..
c WWWWBBB.
d WWWBBB..
e WWWWBB..
f WWWWBB..
g ..WWB...
h ..W.B...
Enter move for colour W (RowCol):   abcdefgh
a ..WW.B..
b ..WWBB..
c WWWWBBB.
d WWWBBB..
e WWWWBB..
f WWWWBB..
g ..WWW...
h ..W.BW..
Enter move for colour B (RowCol):   abcdefgh
a ..WW.B..
b ..WWBB..
c WWWWBBB.
d WWWBBB..
e WWWBBB..
f WWWBBB..
g ..WBB...
h ..WBBW..
Enter move for colour W (RowCol):   abcdefgh
a ..WW.B..
b ..WWBB..
c WWWWBBB.
d WWWBBB..
e WWWWBB..
f WWWBWB..
g ..WWWW..
h ..WBBW..
Enter move for colour B (RowCol):   abcdefgh
a ..WW.B..
b ..WWBB..
c WWWWBBB.
d WWWBBB..
e WWWWBB..
f WWWBWBB.
g ..WWWB..
h ..WBBW..
Enter move for colour W (RowCol):   abcdefgh
a ..WWWB..
b ..WWWB..
c WWWWWBB.
d WWWBWB..
e WWWWWB..
f WWWBWBB.
g ..WWWB..
h ..WBBW..
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b ..BWWB..
c WWWBWBB.
d WWWBBB..
e WWWWWB..
f WWWBWBB.
g ..WWWB..
h ..WBBW..
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b ..BWWB..
c WWWBWBB.
d WWWBBW..
e WWWWWWW.
f WWWBWWB.
g ..WWWB..
h ..WBBW..
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b ..BWWB..
c WWWBWBB.
d WWWBBW.B
e WWWWWWB.
f WWWBWBB.
g ..WWBB..
h ..WBBW..
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b ..BWWB..
c WWWBWBB.
d WWWBBW.B
e WWWWWWWW
f WWWBWBB.
g ..WWBB..
h ..WBBW..
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b ..BBWB..
c WWWBBBB.
d WWWBBB.B
e WWWWWWBB
f WWWBWBBB
g ..WWBB..
h ..WBBW..
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b ..BBWB..
c WWWBBWB.
d WWWWWWWB
e WWWWWWBB
f WWWBWBBB
g ..WWBB..
h ..WBBW..
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b ..BBWB..
c WWWBBWB.
d WWWWWWWB
e WWWWWWBB
f WWWBWBBB
g .BBBBB..
h ..WBBW..
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b ..BBWB..
c WWWBBWB.
d WWWWWWWB
e WWWWWWWB
f WWWBWWWB
g .BBBBBW.
h ..WBBW..
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b ..BBWB..
c WWWBBWB.
d WWWBWWWB
e WWBWWWWB
f WBWBWWWB
g BBBBBBW.
h ..WBBW..
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b ..BBWB..
c WWWBBWB.
d WWWBWWWB
e WWBWWWWB
f WBWBWWWB
g BBBBBWW.
h ..WBBWW.
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b .BBBWB..
c WBBBBWB.
d WBWBWWWB
e WBBWWWWB
f WBWBWWWB
g BBBBBWW.
h ..WBBWW.
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b .BBBWB..
c WBBBBWB.
d WBWBWWWB
e WBBWWWWB
f WBWBWWWB
g WWBBBWW.
h W.WBBWW.
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b .BBBWB..
c WBBBBWB.
d WBWBWWWB
e WBBWWWWB
f WBWBWWWB
g WBBBBWW.
h WBBBBWW.
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b .BBBWB..
c WBBBBWWW
d WBWBWWWB
e WBBWWWWB
f WBWBWWWB
g WBBBBWW.
h WBBBBWW.
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b .BBBWB..
c WBBBBWWW
d WBWBWWWB
e WBBWBWWB
f WBWBWBWB
g WBBBBWB.
h WBBBBBBB
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b .BBBWB..
c WBBBBWWW
d WBWBWWWW
e WBBWBWWW
f WBWBWBWW
g WBBBBWWW
h WBBBBBBB
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b .BBBWBB.
c WBBBBBBW
d WBWBBWBW
e WBBBBWBW
f WBBBWBBW
g WBBBBWBW
h WBBBBBBB
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB.W
b .BBBWBW.
c WBBBBWBW
d WBWBWWBW
e WBBWBWBW
f WBWBWBBW
g WWBBBWBW
h WBBBBBBB
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB.W
b .BBBWBBB
c WBBBBWBB
d WBWBWWBB
e WBBWBWBB
f WBWBWBBB
g WWBBBWBB
h WBBBBBBB
Enter move for colour W (RowCol):   abcdefgh
a .BBBBBWW
b .BBBWWBB
c WBBBWWBB
d WBWWWWBB
e WBWWBWBB
f WWWBWBBB
g WWBBBWBB
h WBBBBBBB
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b .WBBWWBB
c WBWBWWBB
d WBWWWWBB
e WBWWBWBB
f WWWBWBBB
g WWBBBWBB
h WBBBBBBB
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b BBBBWWBB
c WBWBWWBB
d WBWWWWBB
e WBWWBWBB
f WWWBWBBB
g WWBBBWBB
h WBBBBBBB
W player has no valid move.
B player has no valid move.
Black wins by 2 points!
